src/crimson/os/seastore/onode.h

```diff
@@ -31,13 +31,28 @@ struct onode_layout_t {
   ceph_le32 size{0};
   ceph_le32 oi_size{0};
   ceph_le32 ss_size{0};
-  omap_root_le_t omap_root;
+  omap_root_le_t omap_root_large_leaf;
+  omap_root_le_t omap_root_small_leaf;
   omap_root_le_t xattr_root;
 
   object_data_le_t object_data;
 
   char oi[MAX_OI_LENGTH] = {0};
   char ss[MAX_SS_LENGTH] = {0};
+
+  const omap_root_le_t& get_omap_root(omap_type_t type) const {
```

Comment:
Why `xattr_root` isn't included in this method? The 3 omap roots should be symmetric, right?